### PR TITLE
 Cellular.CapabilityVoice and Cellular.CapabilityData props

### PR DESCRIPTION
**Pull request by deztructor**

### CODE REVIEW

File: src/ofono/provider_ofono.cpp [hunk 1]
```diff
@@ -221,7 +223,8 @@ static constexpr const char *property_names[] = {
     "MMSContext",
     "DataRoamingAllowed",
     "GPRSAttached",
-    "VoiceCall"
```
Comment (thp) [inline src/ofono/provider_ofono.cpp]: Is anything depending on this being `Cellular.VoiceCall`?

Comment (deztructor) [inline src/ofono/provider_ofono.cpp]: No, afaik nobody uses it yet, so I just renamed until someboby are going to do it :)

**Comment by VDVsx** [comment]
LGTM, but please separate the sensors stuff into a separated commit before merge.

**Comment by deztructor** [comment]
@VDVx thanks, done

**Comment by monich** [comment]
As far as I know, statefs/ContextKit is allowed in harbour. If that's the case, all changes must be backward compatible.

**Comment by deztructor** [comment]
@monich Public Sailfish OS release 1.1.4.29 includes provider 0.2.68, so VoiceCall property appeared in 0.2.72 has not yet a chance to be used by community apps.

**Comment by monich** [comment]
OK then.

**Pull request merged**